app/src/main/java/cm/aptoide/pt/app/view/AppCoinsInfoFragment.java

```diff
@@ -169,11 +220,99 @@ private void setupToolbar() {
     String installState = getResources().getString(R.string.appview_button_install);
     if (isInstalled) {
       installButton.setText(getResources().getString(R.string.appview_button_open));
+      bottomInstallButton.setText(getResources().getString(R.string.appview_button_open));
     } else {
       installButton.setText(installState);
+      bottomInstallButton.setText(installState);
     }
   }
 
+  @Override public void startCatappultDevWebView() {
+    CustomTabsHelper.getInstance()
+        .openInChromeCustomTab("https://catappult.io/", getContext(), theme);
+  }
+
+  @Override public Observable<ScrollEvent> appItemVisibilityChanged() {
+    return RxNestedScrollView.scrollChangeEvents(scrollView)
+        .flatMap(viewScrollChangeEvent -> Observable.just(viewScrollChangeEvent)
+            .map(scrollDown -> isAppItemShown())
+            .map(isItemShown -> new ScrollEvent(
+                isScrollDown(viewScrollChangeEvent.oldScrollY(), viewScrollChangeEvent.scrollY()),
```

Comment:
When scrolling up, the item with the app will be hidden on my device (as when I open this fragment). I see the bottom card on open but not when scrolling back up. Could it be because we're just listening for scroll down events here?